account_financial_risk/models/account_invoice.py

```diff
@@ -11,17 +11,28 @@ def risk_exception_msg(self):
         self.ensure_one()
         partner = self.partner_id.commercial_partner_id
         exception_msg = ""
+        if self.company_id.currency_id == partner.risk_currency_id:
+            risk_amount = self.amount_total_company_signed
+        else:
+            risk_amount = self.env["res.currency"].with_context(
+                date=self.date_invoice or
+                fields.Date.context_today(self))._compute(
+                self.company_id.currency_id,
+                partner.risk_currency_id,
+                self.amount_total_company_signed,
+                round=False,
```

Comment:
Are you sure you don't want to round the amount?

Reply:
Why would I want to round it?